Clarify vector index dimension best practice (#1513)
neo4j/docs-cypher: modules/ROOT/pages/indexes/syntax.adoc

[OPTIONS "{" option: value[, ...] "}"]
----

> label:new[Introduced in Neo4j 2026.01] Vector indexes support multiple labels, multiple relationship types, and additional properties for filtering. A vector index can index only one vector property, and the optional `WITH` clause adds non-vector properties for filtering; it does not create a composite vector index over multiple vector properties.

Reviewer: ```suggestion
As of Neo4j 2026.01, vector indexes support multiple labels, multiple relationship types, and additional properties for filtering.
A vector index can index only one vector property, and the optional `WITH` clause adds non-vector properties for filtering; it does not create a composite vector index over multiple vector properties.
```
